src/utils/hooks/clusteringWorkRequest.js

```diff
@@ -2,23 +2,25 @@ const constants = require('../../api/general-services/pipeline-manage/constants'
 const workRequestBuilder = require('../workRequestBuilder');
```

Comment:
There is already a folder 'customHooks' inside utils for this

Reply:
Is that in the UI?